src/interface/mod.rs

```diff
@@ -20,8 +20,39 @@
 //! type OledDisplay =
 //!   GraphicsMode<I2cInterface<I2c<I2C1, (PB8<Alternate<OpenDrain>>, PB9<Alternate<OpenDrain>>)>>>;
 //! ```
+//!
+//! [Example](https://github.com/jamwaffles/sh1106/blob/master/examples/blinky_i2c.rs)
```

Comment:
Hmm, not sure where this should link to. Maybe just get rid of it if there are no examples with a big typedef?

Reply:
Oh, those are probably remnants from the commit I reverted. Is it ok to just remove that link and the one below?

Reply:
Yep, they can both go 👌

Reply:
Done.